doc/specs/Keyboard-Selection.md

```diff
@@ -63,6 +68,39 @@ Every combination of the `Direction` and `SelectionExpansionMode` will map to a
 **NOTE**: If `copyOnSelect` is enabled, we need to make sure we update the clipboard on every change in selection.
 
 
+
+### Mark Mode
+Mark Mode is a mode where the user can begin a selection using only the keyboard. It can be cycled into an enabled/disabled state through a user keybinding (`toggleMarkMode`).

```

Comment:
Just to make sure, when you say _only_ the keyboard, that means mouse clicks and drags are ignored?